change/@graphitation-graphql-codegen-typescript-typemap-plugin-4db16247-8e9c-48df-aa31-cd602e20f366.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

Comment:
Since this package has major version as `0` now, maybe this should be `minor` as that would be treated as major change by `^` range?